tutorials/tutorial-hello-timescale.md

```diff
@@ -0,0 +1,798 @@
+- Each airport ride has on average the same number of passengers per trip (~1.7 passengers per trip).
+- The trip distances are roughly the same 16-17 miles.
+- JFK is about 30% cheaper, most likely because of NJ tunnel and highway tolls.
+- Newark trips are 22% (10 min) shorter.
+
+This data is useful not just for city planners, but also for airport travellers 
+and tourism organizations like the NYC Tourism Bureau. For example, a tourism 
+organization could recommend cost-conscious travelers who’d rather not fork out 
+$84 for a ride to Newark to use public transport instead, like the NJ Transit 
+train from Penn Station ($15.25 for an adult ticket). Similarly, they could 
+recommend those travelling to JFK airport, and who are weary of heavy traffic, 
+to take the subway and airtrain instead, for just $7.50.
+
+Moreover, you could also make recommendations for those flying out of 
+New York City about which airport to choose. For example, from the data above, 
+we can recommend those travellers who think they’d be in a rush and who don’t 
+mind paying a little extra to consider flying out of Newark over JFK.
+
+If you’ve made it this far, you’ve successfully completed Mission 2 and now 
+have a basic understanding of how to analyze time-series data using TimescaleDB!
+
+### Mission 3: Monitoring
+
+We can also use the time-series data from taxi rides for monitoring 
+the ride’s current status.
+
+>:WARNING: A more realistic setup would involve creating a data pipeline that streams sensor data directly from the cars into TimescaleDB. However, we will use the January 2016 data to illustrate the underlying principles that are applicable regardless of setup.
+
+#### How many rides took place every 5 minutes for the first day of 2016?
+It’s January 1st 2016. NYC riders have celebrated New Year’s Eve, and using taxi 
+cabs to travel to and from their first gathering of the new year.
+
+The first thing you may like to know is how many rides have recently taken 
+place. We can approximate that by counting the number of rides that were 
+completed on the first day of 2016, in 5 minute intervals.
+
+While it's easy to count how many rides took place, there is no easy way 
+to segment data by 5 minute time intervals in PostgreSQL. As a result, we 
+will need to use a query similar to the query below: 
+
+```sql
+-- Vanilla Postgres query for num rides every 5 minutes
+SELECT
+  EXTRACT(hour from pickup_datetime) as hours,
+  trunc(EXTRACT(minute from pickup_datetime) / 5)*5 AS five_mins,
+  COUNT(*)
+FROM rides
+WHERE pickup_datetime < '2016-01-02 00:00'
+GROUP BY hours, five_mins;
+```
+
+It may not be immediately clear why the above query returns rides segmented 
+by 5 minute buckets, so let’s examine it more closely, using the sample time 
+of 08:49:00.
+
+>:TIP: To see how Timescale simplifies common time-series analysis tasks, like segmenting data by arbitrary time intervals, you can skip ahead to the next query.
```

Comment:
add a link here (on 'skip ahead') to an (also added) anchor below to the next query

Reply:
Deleted this tip altogether. It doesn't really make sense anyway.